sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeCustomFormsOptions.cs

```diff
@@ -1,19 +1,18 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-namespace Azure.AI.FormRecognizer.Models
+namespace Azure.AI.FormRecognizer
 {
     /// <summary>
-    /// The set of options that can be specified when calling a recognition method in
-    /// a <see cref="FormRecognizerClient" /> instance to configure the behavior of the
-    /// request.
+    /// The set of options that can be specified when calling a recognize custom form method
+    /// to configure the behavior of the request.
     /// </summary>
-    public class RecognizeOptions
+    public class RecognizeCustomFormsOptions
```

Comment:
@maririos Was curious if we should match this with Content options naming and keep it as `RecognizeCustomFormOptions` or was the idea to keep it consistent with method naming?

Reply:
yes. consistency with method naming as we use `operation` in both method and options name